memento/src/main/java/com/alexstyl/specialdates/date/Date.kt

```diff
@@ -42,6 +42,8 @@ data class Date(private val localDate: LocalDate, private val yearOptional: Opti
 
     fun hasNoYear(): Boolean = !yearOptional.isPresent
 
+    fun daysInCurrentMonth(): Int = localDate.dayOfMonth().maximumValue
```

Comment:
You can further improve this by having this be a `val` instead. The way you do this in Kotlin is:

```
val daysInCurrentMonth: Int
        get() = localDate.dayOfMonth().maximumValue
```

This is an enhancement, it's up to you if you want to improve it or not :)